conf/nonclangable.conf

```diff
@@ -398,6 +397,7 @@ COMPILER_RT:pn-mpich:toolchain-clang:x86-64 = "-rtlib=libgcc --unwindlib=libgcc"
 COMPILER_RT:pn-aufs-util:libc-glibc:toolchain-clang:x86-64 = "-rtlib=libgcc --unwindlib=libgcc"
 COMPILER_RT:pn-libhugetlbfs:libc-glibc:toolchain-clang:x86-64 = "-rtlib=libgcc --unwindlib=libgcc"
 COMPILER_RT:pn-tsocks:libc-glibc:toolchain-clang:x86-64 = "-rtlib=libgcc --unwindlib=libgcc"
+COMPILER_RT:pn-ostree:toolchain-clang:libc-glibc = "-rtlib=libgcc --unwindlib=libgcc"
```

Comment:
I have used and we only need it for x86-64, for reference I https://github.com/foundriesio/meta-lmp/commit/0abb08c83aee7027711f538e29a5ff16dc334304

Reply:
nice !. Since you have done this and perhaps tested it a bit longer, would you mind proposing it here ? I would like to include those changes instead